CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
 # Change Log
 
-## [v12.15.18](https://github.com/chef/chef/tree/v12.15.18) (2016-10-07)
-[Full Changelog](https://github.com/chef/chef/compare/v12.14.89...v12.15.18)
+## [v12.15.19](https://github.com/chef/chef/tree/v12.15.19) (2016-10-07)
+[Full Changelog](https://github.com/chef/chef/compare/v12.14.89...v12.15.19)
 
 **Enhancements:**
 
-- Adding support for rfc 62 exit code 213 [\#5428](https://github.com/chef/chef/pull/5428) ([jeremymv2](https://github.com/jeremymv2))
+- Adding support for rfc 62 exit code 213 (Chef upgrades) [\#5428](https://github.com/chef/chef/pull/5428) ([jeremymv2](https://github.com/jeremymv2))
```

Comment:
I'll cherry-pick the changelog notes into a different PR and land them on master today.